### ISSUE TITLE
 the algorithm not converge when you run it with default parameters

### ISSUE THREAD

**Issue by yzlicloud** [crashmoon/MAML-Pytorch-Multi-GPUs#3]
The algorithm not converge when i run it with default parameters. the loss and accuracy not chang during the whole training process, i use the default parameter , i really don't why
